services/todoAPI/test/test_service.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+from .. import TodoistService, TodoistRemote, TodoistJSONRemote
+from ...preferences.pref_service import PrefService, PrefJSONRemote
```

Comment:
This does probably work, but `from services.preferences import PrefService, PrefJSONRemote` will do same job, but does not rely on a relative path.

Reply:
done